ext/standard/tests/file/stat_variation6-win32.phpt

```diff
@@ -32,11 +32,12 @@ fclose($file_handle);
 
 // checking stat() on file
 echo "\n*** Testing stat() on file with miscelleneous file permission and content ***\n";
+clearstatcache();
+sleep(1);
```

Comment:
This sleep should go below the stat() call. Also there is similar code further below, where we should also drop the sleep(2) and instead add a sleep(1) between stat() and chmod().